src/index.html

```diff
@@ -6,9 +6,351 @@
+                certifiate, degree or course in a way that’s easy for readers to
+                scan quickly.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Hobbies & Interests Slider-->
+      <div class="slider fade">
+        <div class="top">
+          <div class="title">
+            <h2>HOBBIES & INTERESTS</h2>
+          </div>
+          <div class="media">
+            <a href="">DOWNLOAD MY FULL RESUME</a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-volleyball-ball"></i>
+              <p>Sports</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-theater-masks"></i>
+              <p>Thratre</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-utensils"></i>
+              <p>BBQ</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-glass-cheers"></i>
+              <p>Party</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-cookie"></i>
+              <p>Cooking</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="dot-sections">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+        <span class="dot" onclick="currentSlide(4)"></span>
+      </div>
+    </div>
+
+    <!-- Skill Section-->
+    <div class="skills-section">
+      <h2>SKILLS</h2>
+      <div class="skill">
+        <p class="skill_name">Microsoft Excel</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Microsoft Word</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Adobe</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">PowerPoint</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">French</p>
+        <p class="skill_level">Native</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Spanish</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+    </div>
+
+    <!-- Contacts Section-->
+    <div class="contacts-section">
+      <div class="id-section">
+        <div class="title">
+          <h2>QUICK ID</h2>
+        </div>
+        <div class="border"></div>
```

Comment:
Might use :after on the h2 to achieve the same red divider effect.
This is adding redundant html just trying to get the style, which should be avoid.